src/v2/manifest/manifest_schema2.rs

```diff
@@ -9,6 +14,8 @@ pub struct ManifestSchema2 {
     media_type: String,
     config: Config,
     layers: Vec<S2Layer>,
+    #[serde(skip)]
+    config_blob: Option<ConfigBlob>,
```

Comment:
Why is this field skipped?

Reply:
It's not part of any manifest spec. The intention is to fill this option in with the `fetch_config_blob` method.

Reply:
Mmhhh, that smells quite a bit as we are now mixing together data structures with external specification and internal data-types.
Would it be possible to keep the `ManifestSchema2` and `ConfigBlob` separate and have a super-type which embeds both?

Reply:
Probably so ;-) Would you prefer that?

Reply:
Yes, if the resulting API does not turn into a spaghetti dish of types. I think it will help in the longer to keep the external data-structure as easy to cross-check as possible.

Reply:
I introduced a container type which embeds the manifest and the blob. It looks reasonable to me. PTAL